interface/reports/appointments_report.php

```diff
@@ -315,10 +315,10 @@ class='datepicker form-control'
                                 <a href='#' class='btn btn-default btn-print' id='printbutton'>
                                     <?php echo xlt('Print'); ?>
                                 </a>
-                                <a href='#' class='btn btn-default btn-transmit' onclick='window.open("../patient_file/printed_fee_sheet.php?fill=2","_blank")' onsubmit='return top.restoreSession()'>
+                                <a href='#' class='btn btn-default btn-transmit' onclick='window.open("../patient_file/printed_fee_sheet.php?fill=2", rel="noopener" "_blank")' onsubmit='return top.restoreSession()'>
```

Comment:
Are you sure this will work?
https://stackoverflow.com/questions/46147949/using-rel-noopener-in-window-open

Reply:
Thank you for pointing this out!
It's been removed!